docs/_posts/2016-01-08-A-implies-B-does-not-imply-B-implies-A.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "(A => B) !=> (B => A)"
+author: jimfb
+---
+
+The documentation for `componentWillReceiveProps` states that `componentWillReceiveProps` will be invoked when the props change as the result of a rerender. Some inexperienced users assume this means "if `componentWillReceiveProps` is called, then the props must have changed", but that conclusion is logically incorrect.
```

Comment:
s/inexperienced users/people/

I'm sure plenty of people who have been using React for a while make that assumption as well. I don't see a good reason for qualifying who those people are.